### DISCUSSION TITLE
Difference between two "not found" messages

### DISCUSSION THREAD

**Discussion by SvbZ3r0** [cross-seed/cross-seed#531] [Q&A] [answered]
What is the difference between:
```
info: [server] Received request
info: [torznab] Searching
info: [server] Found 0 torrents
```
and 
```
info: [server] Received request
error: could not find a torrent
```
![image](https://github.com/cross-seed/cross-seed/assets/57280279/2f279133-b683-4fe8-acbf-0f17d0e25d5d)

**Comment by zakkarry** [comment, marked as answer]
> info: [server] Received request
> info: [torznab] Searching
> info: [server] Found 0 torrents

This is when a torrent was sent for searching (you use the name method here) and no matches were found during the search.

> info: [server] Received request
> error: could not find a torrent

This starts the same way, searching for a torrent by name in your case, but there was no torrent file in the torrentDir with the name provided.

In this case, it seems like you may have an extra space on the end of your torrent search term.

I'd recommend using infoHash rather than name. Although it still works with name.

https://www.cross-seed.org/docs/basics/daemon#set-up-automatic-searches-for-finished-downloads

**Reply by SvbZ3r0** [reply]
Thanks for the quick response. I saw these in the logs after qBittorrent sends a signal after downloading.

I tried replicating it by manually copying the torrent name and POSTing it myself, but it looked completely random when I got which result. The same torrent name would sometimes give the first response, and sometimes the second. Maybe I messed up the names.

Will check out the hash method, thanks!
